zebrad/src/commands.rs

```diff
@@ -143,3 +128,132 @@ impl Configurable<ZebradConfig> for ZebradCmd {
         }
     }
 }
+
+/// Toplevel entrypoint command.
```

Comment:
> Moving this code from entry_point.rs and updating it at the same time makes it harder to review.

Moved into its own file in https://github.com/ZcashFoundation/zebra/pull/6801/commits/82fb931bfd5d76c90a5dcd639b5804a72301793b.

Part of it is because of updates to the `Command` trait and part of it is from the migration guide:

```rust
/// Entry point for the application. It needs to be a struct to allow using subcommands!
#[derive(Command, Debug, Clap)]
#[clap(author, about, version)]
pub struct EntryPoint {
    #[clap(subcommand)]
    // Replace this with you app name
    cmd: AppCmd,

    /// Enable verbose logging
    #[clap(short, long)]
    pub verbose: bool,

    /// Use the specified config file
    #[clap(short, long)]
    pub config: Option<String>,
}

impl Runnable for EntryPoint {
    fn run(&self) {
        self.cmd.run()
    }
}
```

Reply:
This PR changes the path of entry_point.rs, so I'll need to do a manual diff.